### DISCUSSION TITLE
Spellchecking linters

### DISCUSSION THREAD

**Discussion by nikhiloberoi94** [LunarVim/LunarVim#3450] [Q&A] [answered]
Hello,
I am trying to get spell check linters such as "misspell" or "codespell" work with .tex and .md files but am unable to do so. I tried to follow the comments in the config file which tell how to include flake-8 and other such linters but still wasn't able to do so. Any links or suggestions on how I can achieve this?

**Comment by kobzar** [comment, marked as answer]
```
local linters = require("lvim.lsp.null-ls.linters")
linters.setup({
	{ command = "flake8", extra_args = { "--max-line-length", "125" }, filetypes = { "python" } },
	{ command = "luacheck", extra_args = { "--ignore", "112" }, filetypes = { "lua" } },
	-- {
	-- 	command = "eslint_d",
	-- 	-- extra_args = { "--max-line-length", "125" },
	-- 	filetypes = { "javascript" },
	-- },
})
```
But you must understand - all of them must be installed in your system before you add they in config

**Reply by nikhiloberoi94** [reply]
Thanks, it did work. I was previously trying to have the filetype as "latex" but when I shifted it to "tex" it worked. 

```
local linters = require("lvim.lsp.null-ls.linters")
linters.setup ({
  { command = "flake8", filetypes = { "python" } },
  { command = "misspell", filetypes = { "tex" , "markdown" } },
})
```
This is what worked for me. Thanks.
